Readme.md

```diff
@@ -39,18 +40,18 @@ as separate *(sub-tree split)* packages for re-usability. As is the case with [S
 
 ### Current Organization
 
-In the doc folder you'll find [Specifications](doc/specifications/) for most features, including the REST API.
+In the doc folder, you'll find [Specifications](doc/specifications/) for most features.
 
 MVC layer:
-- [eZ/Bundle](eZ/Bundle/) is where you'll find bundles are that are important to expose the functionality of the Backend and MVC layer to Symfony.
-- [eZ/Publish/Core/MVC](eZ/Publish/Core/MVC/) contains the parts that make up the different components extending Symfony.
-- [eZ/Publish/Core/Pagination](eZ/Publish/Core/Pagination/) is a component extending PagerFanta for pagination of eZ Platform search queries.
-- [eZ/Publish/Core/REST](eZ/Publish/Core/REST/) is a component providing REST server and *prototype* of a REST Client.
+- [eZ/Bundle](eZ/Bundle/) - the bundles that are important to expose the functionality of the Backend and MVC layer to Symfony.
+- [eZ/Publish/Core/MVC](eZ/Publish/Core/MVC/) - the parts that make up the different components extending Symfony.
+- [eZ/Publish/Core/Pagination](eZ/Publish/Core/Pagination/) - a component extending PagerFanta for pagination of eZ Platform search queries.
+- [ezplatform-rest](https://github.com/ezsystems/ezplatform-rest) - a separate repository providing REST server and *prototype* of a REST Client.
 
 Backend:
-- [eZ/Publish/API](eZ/Publish/API/) is where you'll find the definition of the stable interfaces for the PHP *Public* API, mainly Content *Repository API*.
-- [eZ/Publish/SPI](eZ/Publish/SPI/)  SPI's are *Service Provider Interfaces*, *not yet frozen*.
-- [eZ/Publish/Core](eZ/Publish/Core/) is where you'll find implementations of both APIs and SPIs; the naming aims to map to name of the interface they implement. For example, `Core\Persistence\Legacy` being implementation of `SPI\Persistence`.
+- [eZ/Publish/API](eZ/Publish/API/) - the definition of stable interfaces for the PHP *Public* API, mainly Content *Repository API*.
+- [eZ/Publish/SPI](eZ/Publish/SPI/) - Service Provider Interfaces, *not yet frozen*.
```

Comment:
It requires expanding a bit:
From my POV the `eZ/Publish/SPI` has two parts:
- `eZ/Publish/SPI/Persistence` is a layer which indeed is not frozen yet, meaning it might change in between releases. Those are persistence interfaces for Storage Engine. They can't be frozen yet, because we wouldn't be able to add features to the API (or rather it would be difficult). _We need to improve this layer so we can actually give SPI BC promise on it_.
- `eZ/Publish/SPI/*` (anything other than `Persistence`) is frozen and has a BC promise of Service Provider Interface, meaning no breaking changes both from consumption and implementation POV.
^ please reword it so it's clear to everyone (and ask me if it's not to you as well ;))

FYI @andrerom - I hope the above is fine with you as well.

Reply:
yes, that is ok from my POV